doc/api/next_api_changes/behavior/18226-ES.rst

```diff
@@ -0,0 +1,8 @@
+Widgets use ``CallbackRegistry`` to save callbacks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `.widgets.Widget` classes that allow event observers now use a
+`.CallbackRegistry` to save callbacks. This provides consistency with canvas
+event callbacks, and fixes some bugs that are already handled there. It also
+means that callbacks are now stored as weak references, which means you must
+keep your own reference to them or they may be garbage collected.
```

Comment:
```suggestion
`.Widget`\s with event observers now use a `.CallbackRegistry` for storing
callbacks. This is consistent with canvas event callbacks, and fixes some
bugs in widget callback handling.

Due to this change, callback methods are now stored as weak references,
which means that you should keep a reference to the associated object.
Otherwise it may be garbage collected.
```